src/containers/Node/NodeStructure/NodeStructure.scss

```diff
@@ -94,6 +94,7 @@
         overflow: auto;
 
         min-width: 200px;
+        max-width: unset;
```

Comment:
Without it `max-width ` from uikit was applied and there wasn't enough space for the content